pyathena/connection.py

```diff
@@ -65,6 +66,7 @@ def __init__(
         cursor_kwargs: Optional[Dict[str, Any]] = None,
         kill_on_interrupt: bool = True,
         session: Optional[Session] = None,
+        config: Optional[config.Config] = config.Config(),
```

Comment:
```suggestion
        config: Optional[config.Config] = None,
```

Reply:
I added the changes, but I also simplified the import
from botocore.config import Config